test/actions/open_poll.test.js

```diff
@@ -72,35 +76,81 @@ describe('the OpenPoll action', () => {
   describe('when the poll with the given title does not exist', () => {
     const poll_title = 'Dolor Sit Amet'
 
-    it('creates a new poll with the given title', done => {
-      const seneca_under_test = senecaUnderTest(seneca, done)
+    describe('normally', () => {
+      it('creates a new poll with the given title', done => {
+        const seneca_under_test = senecaUnderTest(seneca, done)
 
-      countPolls(seneca_under_test)
-        .then(num_polls_initially => {
-          Assert.strictEqual(0, num_polls_initially)
-        })
-        .then(() => messageOpenPoll(
-          seneca_under_test,
-          { fields: { title: poll_title } }
-        ))
-        .then(async (result) => {
-          expect(await countPolls(seneca_under_test)).toEqual(1)
-
-          expect(result).toEqual({
-            ok: true,
-            data: {
-              poll: {
-                id: jasmine.any(String),
-                title: poll_title,
-                created_at: jasmine.any(Date),
-                updated_at: null
+        countPolls(seneca_under_test)
+          .then(num_polls_initially => {
+            Assert.strictEqual(0, num_polls_initially)
+          })
+          .then(() => messageOpenPoll(
+            seneca_under_test,
+            { fields: { title: poll_title } }
+          ))
+          .then(async (result) => {
+            expect(await countPolls(seneca_under_test)).toEqual(1)
+
+            expect(result).toEqual({
+              ok: true,
+              data: {
+                poll: {
+                  id: jasmine.any(String),
+                  title: poll_title,
+                  created_at: jasmine.any(Date),
+                  updated_at: null
+                }
               }
-            }
+            })
+
+            return done()
           })
+          .catch(done)
+      })
+    })
 
-          return done()
+    describe('when bombarded with messages to create a poll with the same title', () => {
+      describe('when the lock is enabled', () => {
+        it('only creates one poll', done => {
```

Comment:
Please focus on this test with `fit`. This line of code will then look for you like so:
```
fit('only creates one poll', done => { // fcs
```

Reply:
Then the test can be run:
`$ npm test`